src/control-plane/backend/lambda/api/model/pipeline.ts

```diff
@@ -480,6 +483,13 @@ export class CPipeline {
     }
   }
 
+  private async _checkExistenceS3Bucket() {
+    const buckets = await listBuckets(this.pipeline.region);
```

Comment:
Should we just get the bucket with a certain name to avoid listing hundreds or thousands of buckets?

Reply:
use GetBucketLocationCommand to check existence of bucket